spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -723,56 +620,6 @@ class PodsProjectGenerator
 
             #--------------------------------------#
 
-            describe 'in symlinked directory' do
-              before do
-                # copy banana-lib to a temp directory, make symlink to this dir
-                @tmpdir = Pathname.new(Dir.mktmpdir)
-                old_path_root = Sandbox::PathList.new(fixture('banana-lib')).root
-                FileUtils.copy_entry(old_path_root.to_s, @tmpdir + 'banana-lib')
-                @symlink_dir = old_path_root.dirname + 'banana-lib-symlinked'
-                FileUtils.remove_entry(@symlink_dir) if File.symlink?(@symlink_dir)
-
-                File.symlink(@tmpdir + 'banana-lib', @symlink_dir.to_s)
-
-                # reset project to use symlinked dir
-                @project = Project.new(config.sandbox.project_path)
-
-                path_list = Sandbox::PathList.new(fixture('banana-lib-symlinked/'))
-                @spec = fixture_spec('banana-lib-symlinked/BananaLib.podspec')
-                file_accessor = Sandbox::FileAccessor.new(path_list, @spec.consumer(:ios))
-
-                @project.add_pod_group('BananaLib', fixture('banana-lib-symlinked/'))
-                group = @project.group_for_spec('BananaLib')
-                file_accessor.source_files.each do |file|
-                  @project.add_file_reference(file, group)
-                end
-                file_accessor.resources.each do |resource|
-                  @project.add_file_reference(resource, group)
-                end
-
-                user_build_configurations = { 'Debug' => :debug, 'Release' => :release }
-                @pod_target = PodTarget.new(config.sandbox, false, user_build_configurations, [], Platform.ios,
-                                            [@spec], [@target_definition], [file_accessor])
-                @installer = PodTargetInstaller.new(config.sandbox, @project, @pod_target)
-              end
-
-              after do
-                FileUtils.remove_entry(@tmpdir) if Dir.exist?(@tmpdir)
-                FileUtils.remove_entry(@symlink_dir) if File.symlink?(@symlink_dir)
-              end
-
-              it 'headers are public if podspec directory is symlinked for static lib' do
```

Comment:
is this included elsewhere? were they bad tests?

Reply:
this one was a bad test and its the only one. It was very artificial so I removed it.